src/Test/AbstractBlockServiceTestCase.php

```diff
@@ -20,10 +20,10 @@
 use Symfony\Component\DependencyInjection\ContainerInterface;
 
 /*
- * NEXT_MAJOR: remove check and extend PHPUnit\Framework\TestCase when dropping support for PHPUnit 4
+ * NEXT_MAJOR: remove check when dropping support for PHPUnit 4
  */
-if (!class_exists('\PHPUnit_Framework_TestCase')) {
-    class_alias('PHPUnit\Framework\TestCase', '\PHPUnit_Framework_TestCase');
+if (!class_exists(TestCase::class)) {
```

Comment:
Wouldn't this fail with a used use statement which isn't available?

Reply:
You can try it. Last time I did, it does not matter if class exists or not

Reply:
https://3v4l.org/NSTLf

Reply:
https://3v4l.org/p36qM